static/js/components/VideoPlayer_test.js

```diff
@@ -111,18 +111,33 @@ describe("VideoPlayer", () => {
           },
           playbackRates: [0.5, 0.75, 1.0, 1.25, 1.5, 2.0, 4.0],
           plugins:       {
-            hlsQualitySelector:        {},
-            videoJsResolutionSwitcher: {
-              default:      "high",
-              dynamicLabel: true
-            }
+            hlsQualitySelector: {}
+          },
+          youtube:    { ytControls: 2 },
+          controlBar: {
+            children: [
+              "playToggle",
+              "volumePanel",
+              "progressControl",
+              "remainingTimeDisplay",
+              "playbackRateMenuButton",
```

Comment:
Add "subsCapsButton", for the subtitle control

Reply:
in what order should it appear in the control bar?

Reply:
between playback speed and quality selector:
![Screen Shot 2020-01-16 at 3 50 08 PM](https://user-images.githubusercontent.com/187676/72561810-f0c61b00-3877-11ea-8c0e-c4a5d55982a1.png)